CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Thank you!
 
 # 0.18.32
 
+* codegen: Fix an issue in which smithy4s-protocol would conflict with its previous versions if they're in the dependencies ([#837](https://github.com/disneystreaming/smithy4s/issues/837)) in [#1672](https://github.com/disneystreaming/smithy4s/pull/1672)
```

Comment:
thought these module prefixes would be a nice hint to the users, about which parts of the library were affected. Changed the others in this release in #1673.

Reply:
I'm not sure I understand what you mean here

Reply:
I'm proposing to start formatting the changelog entries so that they start with a `module:` prefix, it'll help the readers of the release notes.